src/models/service/transactions/summary.rs

```diff
@@ -10,6 +10,7 @@ pub struct TransactionSummary {
     pub tx_status: TransactionStatus,
     pub tx_info: TransactionInfo,
     pub execution_info: Option<ExecutionInfo>,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
why only for `safe_app_info` and not for `execution_info` ... should we be consistent for this and always do it (or never), Might not be important for this issue, but worth aligning on.

Reply:
I just added it to `safe_app_info` because it is more often null, I can do it for `execution_info` too 👍 I don't really oppose to this, but it would be nice to be able to do it at a struct level rather than a field level :(